e2e/test/iothub/FileUploadFaultInjectionTests.cs

```diff
@@ -92,13 +95,11 @@ public async Task FileUploadSuccess_QuotaExceed_Amqp()
             TimeSpan operationTimeout = retryDurationInMilliSec == TimeSpan.Zero ? FaultInjection.RecoveryTime : retryDurationInMilliSec;
             deviceClient.OperationTimeoutInMilliseconds = (uint)operationTimeout.TotalMilliseconds;
 
-            using (var fileStreamSource = new FileStream(filename, FileMode.Open, FileAccess.Read))
-            {
-                // UploadToBlobAsync is obsolete, added [Obsolete] attribute to suppress CS0618 message
-                Task fileUploadTask = deviceClient.UploadToBlobAsync(filename, fileStreamSource);
-                Task errorInjectionTask = SendErrorInjectionMessageAsync(deviceClient, faultType, reason, delayInSec, durationInSec);
-                await Task.WhenAll(fileUploadTask, errorInjectionTask).ConfigureAwait(false);
-            }
+            using var fileStreamSource = new FileStream(filename, FileMode.Open, FileAccess.Read);
+            // UploadToBlobAsync is obsolete, added [Obsolete] attribute to suppress CS0618 message
+            Task fileUploadTask = deviceClient.UploadToBlobAsync(filename, fileStreamSource);
```

Comment:
Why not just delete this test since the feature disappeared in v2?

Reply:
It is deleted in v2.

Reply:
The test file.